### DISCUSSION TITLE
How to link the original notes section to a better note notebook 如何将原始笔记部分链接到笔记本

### DISCUSSION THREAD

**Discussion by Liqu0012** [windingwind/zotero-better-notes#752] [Q&A] [answered]
For example:

I have an equation (1), and I have taken notes on the derivation of this equation but would like to have the original text highlighted or tagged so that the next time I read the paper, I am reminded that I have taken notes on this section and can use the link to go directly to it.

How can I do it?

例如我有一个公式（1）,我对这个公式的推导进行了笔记，但是想让原文有高亮或者标记能让我下次阅读论文时提醒我这部分有我的笔记并且能够使用链接直达

**Comment by windingwind** [comment, marked as answer]
1. Highlight in PDF, 
2. Create a note from the highlight annotation,

![image](https://github.com/windingwind/zotero-better-notes/assets/33902321/d8b8b388-fa64-4800-8a69-694f922cab64)

3. Write in that note.

Click the icon again to re-open the note.

or:

Copy/paste the note link to the specific line/section.

![image](https://github.com/windingwind/zotero-better-notes/assets/33902321/cf5d4528-a246-4089-a25e-b36eaee1f046)
